about.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <link rel="stylesheet" href="./about.css" />
+    <title>Microsoft</title>
+  </head>
+  <body>
+    <header class="desktop">
+      <div class="sub-header">
+        <div></div>
+        <ul>
+          <li>
+            <img
+              class="social-icon"
+              src="../Capstone-project/images/home/facebook.svg"
+              alt="facebook"
+            />
+          </li>
+          <li>
+            <img
+              class="social-icon"
+              src="../Capstone-project/images/home/twitter.svg"
+              alt="twitter"
+            />
+          </li>
+          <li><a href="#" class="top-header">English</a></li>
+          <li><a href="#" class="top-header">My page</a></li>
+          <li><a href="#" class="top-header">Log out</a></li>
+        </ul>
+      </div>
+      <nav>
+        <div class="logo">
+          <img src="../Capstone-project/images/microsoft-logo.svg" alt="logo" />
+        </div>
+        <ul>
+          <li><a class="abt" href="./index.html">Home</a></li>
+          <li><a class="abt" href="./about.html">About</a></li>
+          <li>Programe</li>
+          <li>Join</li>
+          <li>Sponser</li>
+          <li>News</li>
+          <li><a class="highlight" href="">Microsoft</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="hero">
+        <nav class="mobile">
+          <div class="burger">
+            <span></span>
+            <span></span>
+            <span></span>
+          </div>
+          <div class="mobile-menu">
+            <ul>
+              <li><a href="../index.html">Home</a></li>
```

Comment:
- [ ] Please, I got 404 error when I tried to navigate to the homepage for the mobile version, perhaps you need to remove one **`.`** from the `href` attribute. ℹ️ 
```suggestion
              <li><a href="./index.html">Home</a></li>
```

- [ ] **Kindly ensure to add appropriate margin to the header section and future section of about.html**.

> |     | Header section                                 | Future Speaker section                                 |     |
> | --- | ----------------------------------------------- | ------------------------------------------------ | --- |
> |     | <img width="288" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/232852560-97ea9d44-8acc-4caf-a078-c07c8aacd171.png"> |<img width="288" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/232852889-d66495d7-fa2d-4440-8bc8-7d802013792f.png">